src/main/java/org/jabref/gui/maintable/BibEntryTableViewModel.java

```diff
@@ -36,6 +40,10 @@
 
 public class BibEntryTableViewModel {
 
+    public static final int FIRST_RANK = 1;
+    public static final int SECOND_RANK = 2;
+    public static final int THIRD_RANK = 3;
+    public static final int FOURTH_RANK = 4;
```

Comment:
Is this better [`f15acdb` (#11510)](https://github.com/JabRef/jabref/pull/11510/commits/f15acdb4ca80efaef28bd386c4844356f325b185)?
I need the integer value for table sorting, also for scrolling to next/prev rank.

Reply:
Can the sorting maybe somehow solved with a custom comparator?

Reply:
Yes, the changes looks better.

Reply:
> Can the sorting maybe somehow solved with a custom comparator?

Using integers is much easier, also for any comparison like in scrollTo, and when comparing for the pseudo-classes.

Reply:
Thought about it again and removed the int value from the enum: [`108a39e` (#11510)](https://github.com/JabRef/jabref/pull/11510/commits/108a39e1bab82fbaa9652039b27ca63ad8e0c764)

Reply:
The table sorting works without a custom comparator because the default comparator for enums is based on the declaration order.

> The natural order implemented by this method is the order in which the constants are declared.

https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/Enum.html#compareTo(java.lang.Enum)

Reply:
looks good!